renovate.json

```diff
@@ -2,14 +2,25 @@
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
     "rebaseWhen": "behind-base-branch",
     "dependencyDashboard": false,
+    "constraints": {
+        "python": "3.8.0"
+    },
     "pip-compile": {
         "fileMatch": [
-            "(^|/)requirements.*\\.in$"
+            "(^|/)requirements.in",
+            "(^|/)requirements-fmt.in",
+            "(^|/)requirements-lint.in",
+            "(^|/)requirements-unit.in",
+            "(^|/)requirements-integration.in"
```

Comment:
I think I know why you have these all listed out and I'm :+1: to that, but can we also put the catch-all pattern of `"(^|/)requirements.*\\.in$"` at the end?  That way if someone adds another environment+requirements file, it'll *probably* have the right thing occur